modules/ROOT/pages/profiles-applications-overview.adoc

```diff
@@ -42,8 +43,8 @@ This profile is also mapped to the xref:user-application-overview.adoc[Bonita Us
 By default, its gives access to the Administrator navigation of Bonita Portal. It includes:
```

Comment:
navigation of Bonita Portal. => navigation in Bonita Applications.

Reply:
Will be handled in another pr